### PR TITLE
Actually add a releases.md

### PR DESCRIPTION
**Pull request by afrittoli**
# Changes

This was meant to be part of https://github.com/tektoncd/cli/pull/1747 but I failed to git add the file 🤦 

Signed-off-by: Andrea Frittoli <andrea.frittoli@gmail.com>

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [x] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [x] Run the code checkers with `make check`
- [x] Regenerate the manpages, docs and go formatting with `make generated`
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/cli/blob/master/CONTRIBUTING.md)
for more details._

# Release Notes
```release-note
NONE
```

### CODE REVIEW

**Comment by afrittoli** [comment]
Thank you @chmouel @vinamra28 for the review on https://github.com/tektoncd/cli/pull/1747
This is the document I had meant to add in there, could I have another review, please? 🙏

**Review by vinamra28 — COMMENTED** [review]
mostly looks good 😄 
/cc @tektoncd/cli-maintainers 
/lgtm

File: releases.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+### v0.26
+
+- **Latest Release**: [v0.26.0][v0-26-0] (2022-09-02) ([docs][v0-26-0-docs])
+- **Initial Release**: [v0.26.0][v0-26-0] (2022-09-02)
+- **End of Life**: 2023-01-08
+- **Patch Releases**: [v0.26.0][v0-26-0]
+
+### v0.25
+
+- **Latest Release**: [v0.25.0][v0-25-0] (2022-07-22) ([docs][v0-25-0-docs])
```
[the remark this answers is not in the document]
Comment (afrittoli) [reply releases.md:55]: Fixed now.

**Review by vdemeester — APPROVED** [review]
/lgtm

**Pull request merged**